Gemfile

```diff
@@ -26,9 +26,6 @@ gem 'whenever', require: false
 
 source 'https://rails-assets.org' do
   gem 'rails-assets-bootstrap', '~> 3.3'
```

Comment:
And finally, this one?

Reply:
Here, I'm OK using [the (real, sanctioned) gem](https://github.com/twbs/bootstrap-sass).

Reply:
Missed your PR text.  FWIW, it's only a "whole project" if you want to upgrade to Bootstrap 4. Sticking with Bootstrap 3 should be pretty easy.

Reply:
The whole reason I'm doing this is to upgrade to bootstrap 4 LOL

Reply:
It can wait. Just thought it might be nice to fully remove rails-assets.

Reply:
I've already done it in another "branch that got too large so I'm breaking it up into smaller PRs" so it should be quick.